src/Utils/RemovableInputField.js

```diff
@@ -1,16 +1,29 @@
 import React, { Component } from 'react';
 
-class InputField extends Component {
+class RemovableInputField extends Component {
     constructor(props) {
         super(props);
         //Function binding
         this.handleInput = this.handleInput.bind(this);
+        this.handleRemove = this.handleRemove.bind(this);
     }
 
     handleInput(event) {
         this.props.onInputChange(this.props.id, event.target.value);
     }
 
+    handleRemove(event) {
+        this.props.onInputChange(this.props.id, '');
+    }
+
+    addRemoveButton() {
+        if (this.props.showRemove){
+            return <button className="input-group-addon" id="remove-addon" onClick={this.handleRemove}>
```

Comment:
Please correct the indentation and wrap the jsx in round brackets. (For a template just look on the render method.)